fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/CascadesPlanner.java

```diff
@@ -973,7 +973,7 @@ protected Object getBindable() {
     }
 
     /**
-     * Adjust Match Task. Attempts to improve an existing partial partial match on a (group, expression) pair
+     * Adjust Match Task. Attempts to improve an existing partial match on a (group, expression) pair
```

Comment:
`group` comes from `GroupExpressionRef` but that is a really poorly chosen name as it clashes with so many other concepts. Just use reference. When we have some time I would like to consolidate `GroupExpressionRef` back into the `ExpressionRef` interface as so far I haven't seen a reason or benefit of having that split (as there is no other implementor; and java typing is horrific around this).

Reply:
I thought the `GroupExpressionRef` corresponds the `Group` concept in Graefe's paper, right? but I agree it is confusing, to me, something like. `EquivalentExpressionsRef` is much clearer.